console/metrics.adoc

```diff
@@ -0,0 +1,52 @@
+include::../partials/attributes.adoc[]
+[id="metrics"] 
+= Flow metrics
+
+The following metrics are available to a Prometheus instance when you enable the flow collector using `skupper init --enable-flow-collector`
+
+flows_total:: For tcp protocol this is the total number of connections. 
+For http or http2 protocol this is the total number of distinct requests made.
+
+octets_total:: The total number of bytes processed by the router.
```

Comment:
Suggesting "The total number of bytes delivered through the skupper network" or something to that effect.